CPAC/func_preproc/func_preproc.py

```diff
@@ -1662,11 +1665,67 @@ def bold_mask_anatomical_based(wf, cfg, strat_pool, pipe_num, opt=None):
     return (wf, outputs)
 
 
+def bold_mask_abcd(wf, cfg, strat_pool, pipe_num, opt=None):
+    '''Resample anatomical brain mask in standard space to get BOLD brain mask in standard space
+    Adapted from DCAN Lab's BOLD mask method from the ABCD pipeline.
+        https://github.com/DCAN-Labs/DCAN-HCP/blob/master/fMRIVolume/scripts/OneStepResampling.sh#L121-L132
+
+    Node Block:
+    {"name": "bold_mask_abcd",
+     "config": ["functional_preproc"],
+     "switch": ["run"],
+     "option_key": ["func_masking", "using"],
+     "option_val": "ABCD",
+     "inputs": ["T1w_template_funcreg",
+                "space-template_desc-brain_T1w",
+                "space-template_desc-T1w_mask"],
+     "outputs": ["space-template_res-bold_desc-brain_T1w",
+                 "space-template_res-bold_desc-T1brain_mask"]}
```

Comment:
"space-template_res-bold_desc-T1brain_mask" is the resulting BOLD mask? I think we can call it desc-bold at this point? Unless I'm missing something.

Reply:
Yes it's used as a BOLD mask but it's actually a resampled T1 mask. Should we call it `space-template_desc-bold_mask`?

Reply:
The way you have it named now is technically correct. You could also keep it that way, as long as later node blocks that use bold masks know to also include that.

And the question is, do we want developers to have to know that whenever they make a node block that uses a bold mask as an input, they'll have to also fold in "space-template_res-bold_desc-T1brain_mask" ?

Reply:
Cool let's call it `space-template_desc-bold_mask`